### PR TITLE
Update Settings Title

### PR DESCRIPTION
**Pull request by neocodegh**
Changed something

### CODE REVIEW

File: SwiftfulSourceControl/SettingsView.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import SwiftUI
 
 struct SettingsView: View {
+    @State private var doSomething: Bool = false
```
Comment (ALI-SAMPSON) [inline SwiftfulSourceControl/SettingsView.swift:11]: Hey this works but we don't use this naming convention. Can you rename this to "isPremium"?

Comment (neocodegh) [inline SwiftfulSourceControl/SettingsView.swift:11]: Can you rename this to "isPremium"?

File: SwiftfulSourceControl/SettingsView.swift [hunk 2]
```diff
@@ -8,6 +8,7 @@
...
     var body: some View {
         Text("Settings!")
```
Comment (neocodegh) [inline SwiftfulSourceControl/SettingsView.swift:13]: Why did you do this?

File: SwiftfulSourceControl/SettingsView.swift (hunk 1 above)
Comment (neocodegh) [reply SwiftfulSourceControl/SettingsView.swift:11]: Sure, I'll change it now.

Comment (neocodegh) [inline SwiftfulSourceControl/SettingsView.swift:11]: Can you change this?

**Review by neocodegh — COMMENTED** [review]
PERFECT!

**Pull request merged**